hal/hal.spec

```diff
@@ -18,7 +18,7 @@
 @ stdcall HalDisableSystemInterrupt(long long)
 @ stdcall HalDisplayString(str)
 @ stdcall HalEnableSystemInterrupt(long long long)
-@ stdcall -arch=i386,arm HalEndSystemInterrupt(long long)
+@ stdcall -arch=i386,arm HalEndSystemInterrupt(long long) #NT use nonstandard parameters calling
```

Comment:
What kind of nonstandard calling it uses?

Reply:
https://jira.reactos.org/browse/CORE-17378?focusedCommentId=127852&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-127852